ocs_ci/ocs/pgsql.py

```diff
@@ -536,3 +544,24 @@ def attach_pgsql_pod_to_claim_pvc(
             self.wait_for_pgbench_status(status=constants.STATUS_COMPLETED, timeout=1800)
 
         return pgsql_obj_list
+
+    def get_postgres_used_file_space(self, pod_obj_list):
+        """
+        Get the used file space on a mount point
+
+        Args:
+            pod_obj_list (POD): List of pod objects
+
+        Returns:
+            list: List of pod object
+
+        """
+        # Get the used file space on a mount point
+        for pod_obj in pod_obj_list:
+            filepath = get_file_path(pod_obj, 'pgdata')
+            filespace = pod_obj.exec_cmd_on_pod(
+                command=f"du -sh {filepath}", out_yaml_format=False
+            )
+            filespace = filespace.split()[0]
+            pod_obj.filespace = filespace
```

Comment:
We have an function get_used_space_on_mount_point in ocs_ci/ocs/resources/pod.py. 
Can this be replaced with that?

Reply:
But I see that function is returning used space in percentage though

Reply:
Used space in mount point won't be much difference after each iteration , so here we are not checking for the used space instead checking for the file size that returns in mb.

Reply:
ack